airqo-platform/AirQo-frontend
# added back to top button

#### Summary of Changes (What does this PR do?)
- added a button which takes user back to top of the page
- have verified locally by running
- Closes #844 

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Run **yarn run docker:build** to build the changes into a container
* Run **yarn run docker:cli** to run the container.
* Access [http://localhost:3000](http://localhost:3000/)
* Click the **Top** icon in the bottom right corner.
* You should be taken to the start of the page.

#### What are the relevant tickets?
- #844 

#### Screenshots (optional)
![image](https://user-images.githubusercontent.com/41787587/199530090-fa960a8e-1b42-47c6-a6ce-10784ad2ab6b.png)
